pages/CONTRIBUTING.md

```diff
@@ -280,32 +255,6 @@ required. To format your code locally, run:
 npm run lint
 ```
 
-### Fork or branch?
```

Comment:
This isn't a question most of our content authors should need to think about, but if they're using the GitHub workflow described above, it'll make the right choice for them automatically. Besides, either way will work.